pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-admin-reconciler.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-admin-reconciler
+rules:
+- apiGroups:
+  - rbac.authorization.k8s.io
+  resources:
+  - rolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  resources:
+  - namespaces
+  verbs:
+  - get
+  - list
+  - watch
+  - create
```

Comment:
If I remove namespace create from the dedicated-admin-reconciler the reconcile fails 
```
+ oc auth reconcile -f /tmp/rolebindings -n test2
rolebinding.rbac.authorization.k8s.io "admin" reconciled
Error from server (Forbidden): namespaces is forbidden: User "system:serviceaccount:openshift-infra:dedicated-admin-reconciler" cannot create namespaces at the cluster scope: User "system:serviceaccount:openshift-infra:dedicated-admin-reconciler" cannot create namespaces at the cluster scope
error: command "/tmp/reconcile" exited with status code 1
error: reached maximum error limit of 1, exiting
```

I believe that is because of this rolebinding:
https://github.com/openshift/openshift-azure/pull/570/files#diff-edf11ce7aa7d5bfaeeaec794f36bc69dR9